src/ngraph_builder.cc

```diff
@@ -1351,6 +1351,136 @@ static Status TranslateConv2DBackpropInputOp(
   return Status::OK();
 }
 
+// Translate DepthToSpace op
+static Status TranslateDepthToSpaceOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input));
+
+  // Get the attributes
+  int64 block_size;
+  std::string tf_data_format;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "block_size", &block_size));
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "data_format", &tf_data_format));
+
+  // The input tensor's height and width must be divisible by block_size
```

Comment:
this comment could be moved to where the check is being performed (line 1390)

Reply:
Sure. Thanks for catching this.